code/modules/reagents/chemistry_reagents/other.dm

```diff
@@ -1008,3 +1008,44 @@
 	chemclass = CHEM_CLASS_SPECIAL
 	properties = list(PROPERTY_TRANSFORMATIVE = 4, PROPERTY_NUTRITIOUS = 3, PROPERTY_HEMOGENIC = 1)
 	flags = REAGENT_SCANNABLE
+
+/datum/reagent/acidic_blood_fuel
+	name = "Acidic Blood Fuel"
+	id = "acidic_blood_fuel"
+	description = "An experimental chemical made from basic xeno blood. Very thin and flammable."
+	color = "#dffc00"
+	burncolor = "#dffc00"
+	burn_sprite = "dynamic"
+	properties = list()
+
+/datum/reagent/acidic_blood_fuel/New()
+	properties = list(
```

Comment:
Why set it in constructor and not directly above in the datum def?

Reply:
That's how other napalms and such have done it in the file. I guess it's refactor time!